src/asciidoc/docs/distribute/app-store.adoc

```diff
@@ -0,0 +1,86 @@
+
+You need the following for your App Store listing:
+
+* A 500x500-pixel icon to be displayed with your listing in the App Store.
+* A 1200x630-pixel thumbnail to be displayed when your listing is shared in other social media sites.
+* Up to 10 videos and screenshots of your app.
+* The production URL of your app.
+To get this URL, go to the app's associated *Project Details*, click *View Published Koji*, and copy the URL from the browser's address bar.
+
+To create your App Store listing,
+
+include::{includespath}/steps-dev-portal.adoc[tag=go2-dev-portal-templates]
+
+. Click *New App Template*.
+
+. Give the app template a name.
++
+This name is not published in the App Store; it is used to identify your listing.
+
+. Fill in the *App Template Details*.
++
+IMPORTANT: All texts and images in your listing must meet https://withkoji.com/resources/terms[Koji's Terms of Use] requirements.
++
+* *Icon* – The 500x500-pixel icon displayed with your listing.
+
+* *Name* – The title of your listing.
+Note that the title could be truncated depending on the width of the browser page; therefore, place distinguishing words near the beginning.
+
+* *Description* – Features and benefits of your app.
+
+* *Share Image* – The 1200x630-pixel thumbnail that is displayed when your app listing is shared in other social media sites.
+
+* *Videos and Screenshots* – Up to 10 videos and screenshots of your app.
+
+* *Bundled App* – The production URL of your app.
+
+* *Price to Remix* – The price (in US dollars) that the creator/influencer pays to publish their customized version of your app.
+Customization is free; creators/influencers are charged only when they publish their customized version in their Koji profile.
+You keep 85% of this amount; Koji gets 15%.
+Leave blank to allow the creator/influencer to publish their customized version for free.
+
+* *Transaction Fee Percent* – The percentage you wish to collect from each transaction if your app allows In-App Purchases.
+You keep 60% of this percentage; Koji gets 40%.
+Leave blank to collect the default percentage (15%).
++
+For example, if you set this to 10% and the user paid $10 in the app,
++
+** The creator/influencer who added your app to their profile receives $9.00.
+** You receive $0.60.
+** Koji receives $0.40.
+
+* *This template is open source.* – If enabled, other developers can view and fork your app's source code.
+
+* *This template is intended primarily for developers to fork.* – If enabled, your app listing will be placed in the *For Developers* category of the App Store.
+
+* *Variants* – (Optional) The listing URLs of other variations of the app.
+
+* *Attachments* – (Optional) Links to other sites that provide more information about your app, such as YouTube tutorials, news articles, and product comparisons or reviews.
+
+. Click btn:[Save and Publish].
+
+. https://forms.gle/exqnEVtC45Cizdw56[Fill out and submit the request form] to add your listing to the App Store.
```

Comment:
Love that we’re solving the submission problem here! 🥇

Reply:
It's just an interim solution. I think we should have an in-house page with the withkoji.com domain. I suggested updating our DNS to add a subdomain with a masked redirect, but Luis thinks it's too early to invest that much work on it at this point.

At least it's better than having them contact Luis directly. He'll never be able to delegate that work if we mention him by name.

Reply:
Ha! Yeah, best not to give them his name, email, and Discord user name LOL.

This seems like a good solution for where we are now. I can imagine a future platform feature that enables developers to submit their apps for review directly from the platform, and keeps a queue of submissions and review statuses where reviewers and developers can access them. Something to build in the future